src/scripts/clipperUI/components/previewViewer/previewViewerHeaderComponentBase.tsx

```diff
@@ -38,11 +38,22 @@ export abstract class PreviewViewerHeaderComponentBase<T, P> extends ComponentBa
 		let buttonGroups = this.getControlGroups();
 
 		for (let i = 0; i < buttonGroups.length; i++) {
-			let id = buttonGroups[i].id;
-			let className = buttonGroups[i].className;
+			let currentButtonGroup = buttonGroups[i];
```

Comment:
You may be able to use this shorthand syntax:

const { id, className, role, isAriaSet } = currentButtonGroup